### ISSUE TITLE
`gsutil -m mv` tells me to use `gsutil -m -m`

### ISSUE THREAD

**Issue by vapier** [GoogleCloudPlatform/gsutil#315]
```
$ lsb_release -ricd
Distributor ID: Ubuntu
Description:    Ubuntu 14.04 LTS (upgraded from: Ubuntu 12.04 LTS)
Release:        14.04
Codename:       trusty

$ gsutil version
gsutil version: 4.15

$ gsutil -m mv gs://chromeos-releases/tobesigned/failed/*smaug* gs://chromeos-releases/tobesigned/
Copying gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.248.0,ChromeOS-accessory_rwsig-R45-7132.248.0-smaug.instructions [Content-Type=text/plain]...
Copying     ...ccessory_rwsig-R45-7132.248.0-smaug.instructions: 186 B/186 B
Removing gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.248.0,ChromeOS-accessory_rwsig-R45-7132.248.0-smaug.instructions...
Copying gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.249.0,ChromeOS-accessory_rwsig-R45-7132.249.0-smaug.instructions [Content-Type=text/plain]...
Copying     ...ccessory_rwsig-R45-7132.249.0-smaug.instructions: 186 B/186 B
Removing gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.249.0,ChromeOS-accessory_rwsig-R45-7132.249.0-smaug.instructions...
Copying gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.250.0,ChromeOS-accessory_rwsig-R45-7132.250.0-smaug.instructions [Content-Type=text/plain]...
Copying     ...ccessory_rwsig-R45-7132.250.0-smaug.instructions: 186 B/186 B
Removing gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.250.0,ChromeOS-accessory_rwsig-R45-7132.250.0-smaug.instructions...
Copying gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.251.0,ChromeOS-accessory_rwsig-R45-7132.251.0-smaug.instructions [Content-Type=text/plain]...
Copying     ...ccessory_rwsig-R45-7132.251.0-smaug.instructions: 186 B/186 B
Removing gs://chromeos-releases/tobesigned/failed/50,canary-channel,smaug,7132.251.0,ChromeOS-accessory_rwsig-R45-7132.251.0-smaug.instructions...

==> NOTE: You are performing a sequence of gsutil operations that may
run significantly faster if you instead use gsutil -m -m ... Please
see the -m section under "gsutil help options" for further information
about when gsutil -m can be advantageous.

... more copying lines ...
```

**Comment by thobrla** [comment]
I suspect that the fix for https://github.com/GoogleCloudPlatform/gsutil/issues/297 covers this.

**Comment by thobrla** [comment]
Fixed with https://github.com/GoogleCloudPlatform/gsutil/commit/ea1ac294dbafc476fbcccdcf940f0382f6d99ef2 and released in gsutil 4.16
